docs/commonbeam.rst

```diff
@@ -21,6 +21,49 @@ will have a very small effect on the size of the common beam.
 The implementation in radio-beam is adapted from `a generalized python implementation <https://github.com/minillinim/ellipsoid/blob/master/ellipsoid.py>`_ and `the original matlab version <http://www.mathworks.com/matlabcentral/fileexchange/9542>`_ written by Nima Moshtagh (see accompanying paper `here <http://citeseerx.ist.psu.edu/viewdoc/download?doi=10.1.1.116.7691&rep=rep1&type=pdf>`__).
 
 
+Convolution to a common resolution
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The typical use case for calculating a common beam is to convolve one or more
+data sets to a common resolution.  Using radio-beam with spectral-cube, you
+would first compute the common beam, then convolve all data sets to that beam.
+
+For a varying-resolution spectral cube, one in which there are different beams
+for each channel, the process looks like:
+
+.. python::
```

Comment:
the docs build fails here. I can't remember why this is, but the rest of our docs don't have this block. They just start with the `>>>`